_sass/custom/custom.scss

```diff
@@ -91,6 +91,56 @@ blockquote p {
   overflow-wrap: break-word;
 }
 
+.bpTable td {
+  vertical-align:top;
+}
+
+#bpImage {
+  height:400px;
```

Comment:
It's better to use width instead of height here. Height allows to control aspect ratio, and it gets stretched and a bit distorted in this case.

Reply:
Width's default is auto is it not? So it will maintain the aspect ratio as long as width is not set. This as height avoids any reflow issues if people are linked to later sections. However, on double checking I did notice one of the doc's template rules was messing with the width so I'll quickly fix that

Reply:
>Width's default is auto is it not? So it will maintain the aspect ratio as long as width is not set.

It is, but in the current case, the aspect ratio is maintained because image is stretched to its maximum width by `max-width: 100%` and height is automatically adjusted by `height` set to `auto`. Otherwise, you force 100% of available width and fixed height.

Your fix will work, but in this case you could replace it with `width: 127px;` to achieve the same result.

Reply:
max width actually shrinks it, it was restraining the width not stretching it. Hard coding the height stops reflow painting, while only width wouldn't provide the canvas with the right dimension. Ideally I would define both but this solves that problem regardless

Reply:
If you don't want it to resize on width reduction, then OK, looks good to me